### ISSUE TITLE
Clean up tsec-compile-check output

### ISSUE THREAD

**Issue by engelsdamien** [microsoft/vscode#116459]
See https://github.com/microsoft/vscode/issues/103699#issuecomment-768914531 for context

the tsec-compile-check action currently has a many outputs, making it difficult to notice when new security-sensitive code is added.

by listing all of the current violations in the tsec exemption file we can ensure that the output of tsec-compile-check stays empty unless someone adds code with a new violation.

**Comment by jrieken** [comment]
wow. chrismas-clean

```
jrieken@Johanness-MBP-2 vscode % yarn run tsec-compile-check
yarn run v1.19.1
$ node node_modules/tsec/bin/tsec -p src/tsconfig.tsec.json
✨  Done in 39.06s.
jrieken@Johanness-MBP-2 vscode % 
```

**Comment by jrieken** [comment]
@engelsdamien Thanks for this. I believe it's again time to enable this in the CI pipelines and to bring back the tsserver integration. The only remaining worry is our aggressive TS adoption strategy which can often mean that tsec is facing TS features is doesn't understand yet.

**Comment by jrieken** [comment]
tsec is part of our CI pipeline 🎉 👯

@engelsdamien I also tried to configure the tsserver plugin but I couldn't get it to accept the exemptions list. I have tried with below snippet in `src/tsconfig.json`

```
	"plugins": [
		{
		"name": "tsec",
		"exemptionConfig": "./tsec.exemptions.json"
	}
]
```

**Comment by engelsdamien** [comment]
@uraj for visibility on the tssever plugin issue

**Comment by uraj** [comment]
There is a bug in the plugin. Fix is on the way.

**Comment by uraj** [comment]
The bug in the plugin should have been fixed in 0.1.4.

**Comment by jrieken** [comment]
Thanks. Adopted and enabled
